testdata/project-v2-multigroup/go.mod

```diff
@@ -4,8 +4,6 @@ go 1.13
 
 require (
 	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
```

Comment:
Why a cleanup in the tests is change the go.mod?

Reply:
This two requires are added to go mod when you run the unit tests of the projects. Previously `generate_tstdata.sh` was running the tests, now there is a specific target for running the tests and they are being run in the CI but not when generating the testdata. `test/testdata/generate.sh` just generates it, `test/testdata/check.sh` regenerates it and checks if there was any diff, and `test/testdata.test.sh` performs the tests in the testdata files. So as no tests are being run in generate these two packages are not `go get`ed and therefore not listed in the `go.mod` files.

Reply:
The generate_testdata.sh run make all for all generated project which is what prevent we introduce issues many times. For example, some scaffold issue that will affect a makefile target or missing the binaries to run the make tests target of the projects scaffolds. 

Could you please point out where the make all still been called for all testdata samples in order to allow the merge?

Reply:
I found: 

make test -> call integration-test which will run make https://github.com/kubernetes-sigs/kubebuilder/pull/1933/files#diff-21d992da4bf631674105f51a8e9ddf608cef06e0e53b568489c991e8f34ce32dR58

Reply:
Exactly